src/adapters/mongoose.js

```diff
@@ -143,6 +143,17 @@ module.exports = (model, opts) => {
         // Schema
         return [schemaType(fieldInfo.options.type[0])];
       }
+
+      // NOTICE: Object with `type` reserved keyword.
+      //         See: https://mongoosejs.com/docs/schematypes.html#type-key).
```

Comment:
Is the trailing ")" an error?
```suggestion
      //         See: https://mongoosejs.com/docs/schematypes.html#type-key
```